src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+
+    result: 0,
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
```

Comment:
```suggestion
    subtract: (a, b) => a - b;
```
to make it look simpler :)

Reply:
Thank you! :)